app/mailers/sync_mailer.rb

```diff
@@ -0,0 +1,14 @@
+# app/mailers/sync_mailer.rb
+class SyncMailer < GenericMailer
```

Comment:
Please explain about the mailer in the comments for every type of mailer that you are creating. Here is an example: answer to the question: What is the purpose of SyncMailer?